### DISCUSSION TITLE
Hide the "Like" and "Dislike is Disabled" captions next to the buttons

### DISCUSSION THREAD

**Discussion by TsukiZero** [Anarios/return-youtube-dislike#1076] [Ideas]
It is breaking the action bar badly and none of my CSS fixes can work with it (trying to do style manipulation causes a bad loop)

**Comment by jpa102** [comment]
it will be like this, right?

![image](https://github.com/user-attachments/assets/5e5f2da3-c7de-4a13-8a4e-2362a9ef5b1e)

**Reply by TsukiZero** [reply]
correct!

**Reply by jpa102** [reply]
> (trying to do style manipulation causes a bad loop)

i don't know, but i wrote a simple javascript code for this:

```
document.head.insertAdjacentHTML(
	"afterbegin",
	`
		<style id="hide-like-counts-and-dislike-counts-text">
			/* dislike count renderer */
			dislike-button-view-model > toggle-button-view-model > button-view-model > button > div.yt-spec-button-shape-next__button-text-content {
				display: none !important;
			}
			
			/* like count renderer */
			like-button-view-model > toggle-button-view-model > button-view-model > button > div.yt-spec-button-shape-next__button-text-content {
				display: none !important;
			}
			
			/* unset the margin-right for like & dislike icons */
			dislike-button-view-model > toggle-button-view-model > button-view-model > button > .yt-spec-button-shape-next__icon,
			like-button-view-model > toggle-button-view-model > button-view-model > button > .yt-spec-button-shape-next__icon {
				margin-right: unset !important;
			}
		</style>
	`
);
```

this is the result: (i found a video with hidden likes from my history)

![image](https://github.com/user-attachments/assets/008718a0-0197-462b-9ccc-fbee01f5510d)

without the javascript code: (for some reason, it's displaying with japanese text on my end)

![image](https://github.com/user-attachments/assets/93c68481-3666-4faf-a4d8-b67e65a2be00)

maybe i can try implementing this in the userscript version (i forked the repository months ago)
